### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by fivetran-reneeli**
## PR Overview
**This PR will address the following Issue/Feature:** n/a

**This PR will result in the following new package version:** v0.10.0

Linked to the upstream update https://github.com/fivetran/dbt_shopify_source/pull/75

Will require a full refresh to update the logic coming from dbt-date and dbt-expectations

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

Updates the dependency versions to the most current 

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [ ] dbt run –vars (if applicable)

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [ ] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

Run with a full refresh

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ n/a] DECISIONLOG updates have been updated (if applicable)
- [ ] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [x] docs were regenerated (unless this PR does not include any code or yml updates)

### If you had to summarize this PR in an emoji, which would it be?
 
:dancer:

### CODE REVIEW

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:2]: Can we add a subheader here

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:2]: Additionally can we include the a section here that highlights the tests removed

File: packages.yml
Comment (fivetran-joemarkiewicz) [inline packages.yml:7]: Reminder to update before merge

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
Small request to make an update to the changelog

File: CHANGELOG.md
Comment (fivetran-reneeli) [reply CHANGELOG.md:3]: updated

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:4]: I think this should be in a different subheader. The dependency update is for just remove the dependency, but this should be listed as a general update. 

Also, this should have a PR link

**Review by fivetran-joemarkiewicz — APPROVED** [review]
I just have a small request to more appropriately parse out the CHANGELOG entries into their own subheaders.

Once that is applied then this is good to go! Please remember to only merge this once the source package updates are live. Otherwise all of the buildkite jobs will show as failed.

**Pull request merged**
